golang/cosmos/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/armon/go-metrics v0.4.1
 	github.com/cosmos/cosmos-sdk v0.46.16
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v6 v6.1.2
-	github.com/cosmos/ibc-go/v6 v6.2.1
+	github.com/cosmos/ibc-go/v6 v6.3.0
```

Comment:
Upstream `ibc-go` is at v6.3.1 now.  Could you please change the dependency to that instead?

Reply:
Good call, just bumped to v6.3.1